app/services/curation_concerns/actors/actor_factory.rb

```diff
@@ -9,6 +9,7 @@ def self.build(curation_concern, current_user)
 
       def self.stack_actors(curation_concern)
         [AddToCollectionActor,
+         AddAsMemberOfCollectionsActor,
```

Comment:
My intention was to leave the existing membership functionality for Collections to continue to have member Collections.  But I could just replace `AddToCollectionActor` if that makes more sense.